.github/workflows/ci.yml

```diff
@@ -192,6 +192,19 @@ jobs:
           name: nuget-packages
           path: ./Bin/Release/ILGPU*.${{ steps.package.outputs.version }}.*nupkg
 
+  publish-nuget-pre-release:
+    if: github.ref == 'refs/heads/master'
+    needs: [check-style, build-and-test]
+    runs-on: windows-latest
+    steps:
+      - name: Download NuGet package artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: nuget-packages
+      - name: Publish to FeedzIO
+        shell: bash
```

Comment:
SUGGESTION: Instead of running on `windows-latest` and using `bash` shell, just run on `ubuntu-latest`.